osu.Game/Overlays/UserProfileOverlay.cs

```diff
@@ -30,8 +30,8 @@ public class UserProfileOverlay : FullscreenOverlay
         private ProfileSectionsContainer sectionsContainer;
         private ProfileTabControl tabs;
 
-        private Bindable<RulesetInfo> displayedRuleset;
         private Bindable<User> displayedUser;
```

Comment:
readonly

Reply:
We can't use readonly here because we are resetting them in `Show()`